dev-db/mycli/mycli-1.26.1-r2.ebuild

```diff
@@ -72,3 +72,11 @@ EPYTEST_IGNORE=(
 distutils_enable_tests pytest
 
 PATCHES=( "${FILESDIR}/mycli-1.21.1-fix-test-install.patch" )
+
+src_prepare() {
+	default
+
+	# Remove unnecessary pin, we have paramiko 3.2.0.
+	# https://github.com/dbcli/mycli/commit/eaddc5ca3e208d66fd4f400b90eb76089dd35e4c
+	sed -i -e 's/\(cryptography\) == [0-9.]*/\1 >= 1.0.0/' setup.py || die
```

Comment:
In general, just replace `s:==:>=:` and be done with it. We want the `>=` anyway.

Reply:
Ill try to wean off the habit of overdoing sed :P